src/SqlPersistence/SqlPersistenceStorageSessionExtensions.cs

```diff
@@ -22,7 +22,16 @@ public static ISqlStorageSession SqlPersistenceSession(this SynchronizedStorageS
             return GetSqlStorageSession(session);
         }
 
-        static StorageSession GetSqlStorageSession(this SynchronizedStorageSession session)
+        static ISqlStorageSession GetSqlStorageSession(this SynchronizedStorageSession session)
```

Comment:
couldn't this just be inlined at line 22?

Reply:
Done